tests/end_to_end/README.md

```diff
@@ -0,0 +1,43 @@
+# End-to-end tests
+
+This directory contains end-to-end tests for DepthAI. These tests are designed to run on a real device and test the complete pipeline with camera, neural network, and parsers.
+The tests check if the device is able to run a model with neural network node and parse the output with parser node and return the message. If the error is raised during the process, the test will fail.
+
+## Running the tests on GitHub Actions
+
+You can manually trigger the Github action to run the HIL tests. The is a workflow parameter `additional-parameter` that can be used to specify the desired test. Default is `-all`. The available options are: `-all`, `-p <parser_name>`, `-s <slug_1> <slug_2> ...`, `-nn <path_to_archive_1> <path_to_archive_2> ...`.
```

Comment:
"The is a workflow parameter" -> "There is a ..."?

Reply:
Thanks, fixed in [3ecafa6](https://github.com/luxonis/depthai-nodes/pull/122/commits/3ecafa6cfdb2080d86b0dec9541f8d259f42ba27).